src/components/HdLink.vue

```diff
@@ -49,3 +51,23 @@ export default {
   },
 };
 </script>
+
+<style lang="scss">
```

Comment:
```suggestion
<style lang="scss" scoped>
```

We should scope our component styles.
I think you left it unscoped just to allow the us to use `.link` across the app. But to achieve that I recommend adding the styles directly to the `main.scss` to make it more visible and expected.
You can create a separate file, the same way you did for `typography.scss`, maybe call it `link.scss`.

For now you can just import the file in `main.scss` directly (to avoid breaking changes), but later we can make those functional imports optional.